cms/djangoapps/contentstore/views/course.py

```diff
@@ -829,14 +830,13 @@ def course_info_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             return render_to_response(
                 'course_info.html',
-                {
+                dict({
                     'context_course': course_module,
                     'updates_url': reverse_course_url('course_info_update_handler', course_key),
                     'handouts_locator': course_key.make_usage_key('course_info', 'handouts'),
                     'base_asset_url': StaticContent.get_base_url_path_for_course_assets(course_module.id),
-                    'keywords_supported': get_keywords_supported(),
                     'push_notification_enabled': push_notification_enabled()
-                }
+                }, **get_context_for_course_info_handler())
```

Comment:
@kluo The problem there is that in order to declare a new variable, it would need to be moved up above the `return` statement. Doing so would require changing the indent level of the `dict`, meaning that this entire block would conflict with what edX has.